src/main/js/controls/Bar/helpers/legendHelpers.js

```diff
@@ -47,16 +47,55 @@ const clickHandler = (graphContext, control, config, canvasSVG) => (
             shownTargets.push(item.key);
         }
     };
+    if (config.shownTargets.length === 0) {
+        graphContext.contentKeys.forEach((i) => {
```

Comment:
Would something other than _i_ be a better variable name? e.g. _key_

Reply:
[Fixed](https://github.com/cerner/carbon-graphs/pull/300/commits/595865c1ef97c1cd837971b727d58b1e7d89c5c0#diff-1ecc848586009dc5a2a7976d5f75492eR51-R53)